src/reducers/collapseContext.ts

```diff
@@ -19,7 +27,13 @@ const collapseContext = (state: State) => {
           newPath: unroot([...parentOf(cursor), child]),
         })
       ),
-      archiveThought({ path: cursor }),
+      !preventCursorArchive
+        ? archiveThought({ path: cursor }) :
+        existingThoughtDelete({
+          context: parentOf(pathToContext(cursor)),
```

Comment:
Also here: `pathToContext(simpleCursor)`.

It's not important, but it would be a bit cleaner to store it in a `context` variable to avoid calling `pathToContext` multiple times.

Reply:
Sure.